utils/design-vars.stanza

```diff
@@ -13,3 +18,59 @@ public var PREFERRED-MOUNTING = "smd" ; values in ["smd", "through-hole"]
 public var MIN-CAP-VOLTAGE = 10.0 ; Minimum voltage to allow for capacitors
 ; For consolidation
 ;public var PREFERRED-MANUFACTURERS = ["Yageo", "Panasonic Electronic Components", "Vishay Dale", "TE Connectivity Passive Product"]
+
+
+;------------------------------------------------------------------------------
+; Design variables used to configure land pattern generation/checks
+;------------------------------------------------------------------------------
+
+; Corresponds to IPC 6011
+public defenum DesignProducability:
+  LevelA
+  LevelB
+  LevelC
+
+; The construction type of the design (rigid, flex)
+public defenum DesignConstructionType:
+  Rigid
+  Flex
+
+; The application domain of the design (consumer, automative, aerospace)
```

Comment:
I'm not sure why we need this information for land patterns. I think we should put considerable thought into how to categorize things and what these flags do before adding them. These categories may be too broad to be useful.